NAPS2.Lib/Config/CommonConfig.cs

```diff
@@ -166,4 +166,8 @@ public class CommonConfig
 
     [Common]
     public bool EnableDebugLogging { get; set; }
+    
+    [User]
+    public bool EnableThumbnailText { get; set; }
```

Comment:
Can we call this `ShowPageNumbers` instead?

Reply:
Sure. I'll change it.